auto_landing/.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Remove this file as well. You should add a .gitignore file that allows git to ignore the .vscode folder.